snipssonos/services/music/search_service.py

```diff
@@ -1,5 +1,8 @@
 class MusicSearchService(object):  # TODO : Make this class Abstract
 
+    def set_node_query_builder(self, device_name):
```

Comment:
This should not belong here. We are defining an interface here. We don't care if the query builder is implemented for node, or for spotify or for deezer